### DISCUSSION TITLE
Mimimize to system tray

### DISCUSSION THREAD

**Discussion by Ahmetozefe** [localsend/localsend#16] [Discussion (English, International)] [answered]
It would be amazing to be able to minimize LocalSend to system tray on Windows, so it's always ready to receive files from other devices.

**Comment by Tienisto** [comment, marked as answer]
Hi, this feature will be implemented soon. Created #17 and #18

**Reply by GhosTHaise** [reply]
thanks !

**Reply by ghost** [reply]
thanks you for comment!

**Reply by jeffreyd00** [reply]
reading this makes me very happy!
many kudos to Tienisto for the amazing family of apps!
